indexer/layerscanner.go

```diff
@@ -145,7 +145,10 @@ func (ls *LayerScanner) Scan(ctx context.Context, manifest claircore.Digest, lay
 				return context.Cause(ctx)
 			default:
 			}
-			return ls.scanLayer(ctx, l, s)
+			if err := ls.scanLayer(ctx, l, s); err != nil {
+				return fmt.Errorf("failed to scan layer %s: %w", l.Hash, err)
```

Comment:
```suggestion
				return fmt.Errorf("%s: %w", l.Hash, err)
```

The caller prepends their own context, which I think is something like "failed to scan all layer contents:", so reiterating that a _layer scan_ is what failed seems to add very little?

Reply:
I noticed we also use `%q` in other places for `layer.Hash`s so switched that too